django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -579,11 +579,13 @@ ul.timelist, .timelist li {
 }
 
 .related-widget-wrapper-link {
-    opacity: 0.3;
+    opacity: .6;
+    filter: grayscale(1);
 }
 
 .related-widget-wrapper-link:link {
-    opacity: .8;
+    opacity: 1;
+    filter: grayscale(0);
 }
 
 .related-widget-wrapper-link:link:focus,
```

Comment:
Those hover / focus styles should be removed if we always have full opacity for links.

Reply:
Now have been removed.